src/automations/checkInactivity.js

```diff
@@ -98,25 +99,45 @@ async function checkInactivePullRequest(pullRequest) {
   }
 }
 
-async function scrapeInactiveIssues(references, issues) {
-  const ms = this.cfg.activity.check.limit * 86400000;
-  const ims = this.cfg.activity.check.reminder * 86400000;
+async function extractPRLabels(client, repoOwner, repoName, number) {
```

Comment:
Remember to remove `client` once you bind the client to the function with the `Function.call()` method